ethcore/vm/src/action_params.rs

```diff
@@ -76,7 +76,9 @@ pub struct ActionParams {
 	pub sender: Address,
 	/// Transaction initiator.
 	pub origin: Address,
-	/// Gas paid up front for transaction execution
+	/// Gas paid up front for entire transaction execution.
+	pub entire_gas: U256,
+	/// Gas paid up front for action execution.
```

Comment:
`Gas paid up front for action execution (entire gas minus action opcode and data cost).` ?